packages/server/src/db/defaultData/datasource_bb_default.ts

```diff
@@ -89,9 +90,10 @@ const AUTO_COLUMNS: TableSchema = {
 
 export const DEFAULT_INVENTORY_TABLE_SCHEMA: Table = {
   _id: DEFAULT_INVENTORY_TABLE_ID,
-  type: "internal",
+  type: "table",
```

Comment:
Will this cause any conflict with the existing apps?

Reply:
No - I've updated the frontend to no longer use the `type` and the backend doesn't really use it either - they all depend on the `sourceType` instead now. Old apps will have this added onto them as part of enrichment.

Reply:
So, if nobody is using that, why do we need this field?

Reply:
We don't massively need it - but other documents have this `type` - like `row`, where it is quite important for the internal DB. I'd like to keep this consistent at least - I hadn't realised it was as inconsistent as it was.